butter-cms.json

```diff
@@ -28,56 +28,56 @@
     "Search Features [Autosuggestion]": "No",
     "Search Features [Result Highlighting]": "No",
     "Search Features [Geospatial Search]": "No",
-    "Search Features [Search within uploads (pdf,doc)]": "No",
+    "Search Features [Search within uploads (pdf,doc)]": "Yes",
```

Comment:
https://buttercms.com/kb/media-library

Reply:
I don't see where it is mentioned that it is possible to search within the text content of uploaded documents.

Reply:
Ah, gottcha.  This would remain No for now in this case.  Maybe this Feature could be renamed to [Search within text content of uploads (pdf, doc)]

Reply:
Yes, there are actually a few entries in the list which are not 100% clear. I hope to update the tables in the future to make it less confusing.